<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PETool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>601</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>590</height>
     </size>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>说明</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>是否PE</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string> </string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>入口地址</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string> </string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>文件基址</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string> </string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>文件偏移</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string> </string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>文件大小</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string> </string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>创建时间</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string> </string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>修改时间</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string> </string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>访问时间</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string> </string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Calibri</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIMAGE_DOS_HEADER"/>
   <addaction name="separator"/>
   <addaction name="actionIMAGE_FILE_HEADER"/>
   <addaction name="separator"/>
   <addaction name="actionIMAGE_OPTIONAL_HEADER"/>
   <addaction name="separator"/>
   <addaction name="actionIMAGE_DATA_HEADER"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionIMAGE_DOS_HEADER">
   <property name="text">
    <string>DOS_HEADER</string>
   </property>
   <property name="iconText">
    <string>DOS_HEADER</string>
   </property>
   <property name="toolTip">
    <string>DOS_HEADER</string>
   </property>
  </action>
  <action name="actionIMAGE_FILE_HEADER">
   <property name="text">
    <string>FILE_HEADER</string>
   </property>
  </action>
  <action name="actionIMAGE_OPTIONAL_HEADER">
   <property name="text">
    <string>OPTIONAL_HEADER</string>
   </property>
  </action>
  <action name="actionIMAGE_DATA_HEADER">
   <property name="text">
    <string>DATA_DIRECTORY</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
